src/program.ts

```diff
@@ -1614,7 +1614,8 @@ export class Program extends DiagnosticEmitter {
     element = assert(parent.lookupInSelf(name)); // possibly merged
     var members = declaration.members;
     for (let i = 0, k = members.length; i < k; ++i) {
-      switch (members[i].kind) {
+      let member = members[i];
+      switch (member.kind) {
         case NodeKind.CLASSDECLARATION: {
           this.initializeClass(<ClassDeclaration>members[i], element, queuedExtends, queuedImplements);
```

Comment:
Should use `member` here and in the cases below as well then probably

Reply:
yeah. Good catch!

Reply:
It seems this regress after resolving conflict with upstream

Reply:
fixed